docs/client.rst

```diff
@@ -259,10 +290,10 @@ This implies:
 
 The ``_until_timeout`` suffix function version will perform these actions until the waiting for a new message reaches the timeout.
```

Comment:
Not clear for the user this function finishes when a user data was received. Clarify.

Reply:
Updated